sql/queries.sql

```diff
@@ -362,3 +362,236 @@ CREATE VIEW user_balance
     SELECT payment.user_id, SUM(payment_amount) - SUM(bill_amount) AS balance
     FROM payment, bill
     WHERE payment.user_id = bill.user_id;
+
+
+-- --------------------------------------------------------------------------------------------------------------------------------
+
+** ABDUL'S QUERIES **
+
+#i
+-- CREATING AN EMPLOYER
+INSERT INTO users (user_id, email, password)
+    VALUES (11, 'agohier01@booking.com', 'xpnTTJ81');
+    
+INSERT INTO employer (employer_id, employer_name, description, employer_tier)
+	VALUES (11, 'Mike', 'Front_end developer', '0');
+    
+-- EDITING AN EMPLOYER INFO
+UPDATE employer
+SET description='Full Stack Developer'
+WHERE employer_id=11;
+
+-- DISPLAYING AN EMPLOYER
+SELECT *
+FROM employer
+WHERE employer_id=11;
+
+-- DELETING AN EMPLOYER
+DELETE FROM users
+WHERE user_id=11;
+
+DELETE FROM employer
+WHERE employer_id=11;
+
+-- -----------------------------------------------------------------------
+#ii 
+-- CREATING A CATEGORY BY AN EMPLOYER
+INSERT INTO employer (employer_id, employer_name, description, employer_tier)
+	VALUES (6, 'Mike', 'Front_end developer', '0');
+INSERT INTO employer_category(employer_id, category)
+	VALUES (6, 'IT');
+    
+-- EDITING A CATEGORY BY AN EMPLOYER
+UPDATE employer_category
+SET category='Tech'
+WHERE employer_id=6;
+
+-- DISPLAYING A CATEGORY BY AN EMPLOYER
+SELECT *
+FROM employer_category
+WHERE employer_id=6;
+
+-- DELETING A CATEGORY BY AN EMPLOYER
+DELETE FROM employer_category
+WHERE employer_id=6;
+-- -----------------------------------------------------------------------
+#iii
+-- POSTING A NEW JOB BY AN EMPLOYER
+
+INSERT INTO posting(employer_id, posting_id, title, description, number_of_openings)
+	VALUES (1, 4, 'Construction worker', 'Infrastructure speaclist', 2);
+    
+-- -----------------------------------------------------------------------
+#iv
+-- Providing a job offer for an employee by an employer.
+
+UPDATE application
+SET offer_time = CURRENT_TIMESTAMP
+WHERE application_id = 1 AND job_seeker_id = 6
+    AND employer_id=1;
+
+-- -----------------------------------------------------------------------
+#v 
+-- Report of a posted job by an employer 
+
+SELECT posting.title, posting.description, date_posted, app.job_seeker_id, status
+FROM posting p, application app
+WHERE  p.posting_id=app.posting_id AND
+p.posting_id=1 AND employer_id=1;
+
+-- -----------------------------------------------------------------------
+#vi 
+-- Report of posted jobs by an employer during a specific period of time (Job
+-- title, date posted, short description of the job up to 50 characters, number
+-- of needed employees to the post, number of applied jobs to the post,
+-- number of accepted offers).
+
+SELECT posting.title, posting.date_posted, posting.description, posting.number_of_openings,
+ COUNT(application.application_id), COUNT(application.status)
```

Comment:
We need to count application_id and job_seeker_id together because that's what determines a unique application; in the second COUNT, application.status needs to be equated with 'accepted'.